INFO     cardano_node_tests.tests.conftest:conftest.py:136 Changed CWD to '/home/martink/Source/repos/cardano-node-tests4/tmp/pytest-of-martink/pytest-56/popen-gw2'.
INFO     cardano_node_tests.utils.cluster_management:cluster_management.py:443 Stopping cluster with `/home/martink/Source/repos/cardano-node-tests4/tmp/pytest-of-martink/pytest-56/cluster4/startup_files/kwdfmmiv/stop-cluster-hfc`.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests4/.cli_coverage/cli_coverage_script_220807_010001481_fqsd.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests4/tmp/pytest-of-martink/pytest-56/cluster_artifacts/state-cluster4_uryoexik'.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:353 Starting cluster with `/home/martink/Source/repos/cardano-node-tests4/tmp/pytest-of-martink/pytest-56/cluster4/startup_files/kwdfmmiv/start-cluster-hfc`.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:355 Cluster started.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:377 Performing 'restart' action on services ['nodes:pool2', 'nodes:pool3'].
INFO     cardano_node_tests.tests.test_kes:test_kes.py:245 2022-08-07 01:17:19.041911: Waiting for 200 sec for KES expiration.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:249 2022-08-07 01:20:39.141376: KES expired (?); tip: '{'block': 569, 'epoch': 6, 'era': 'Alonzo', 'hash': 'd045982acc39d2f69d585dc2263f6ef63bbc1a9594cf94d1a5f4a11c6adb48fb', 'slot': 5251, 'syncProgress': 100.0}'.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:130 2022-08-07 01:20:39.177716: Waiting for 1 new epoch(s).
INFO     cardano_clusterlib.clusterlib_klass:clusterlib_klass.py:3045 Waiting for 170.80 sec for slot no 6105.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:133 2022-08-07 01:23:30.144408: Waiting for the end of current epoch.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:377 Performing 'restart' action on services ['nodes:pool2', 'nodes:pool3'].
INFO     cardano_node_tests.tests.test_kes:test_kes.py:267 2022-08-07 01:26:35.848876: Waiting 120 secs to make sure the expected errors make it to log files.